.dev_scripts/benchmark_regression.py

```diff
@@ -0,0 +1,311 @@
+        model_cfg_infer = [
+            cfg['model_list']['inference'][i] for i in range(ori_len)
+            if cfg['model_list']['inference'][i]['priority'] <= args.priority
+        ]
+        num_infer_task = len(model_cfg_infer)
+        num_infer_win = math.ceil(num_infer_task / args.panes_per_window)
+    else:
+        num_infer_task = 0
+        num_infer_win = 0
+
+    if 'train' in cfg['model_list'].keys():
+        ori_len = len(cfg['model_list']['train'])
+        model_cfg_train = [
+            cfg['model_list']['train'][i] for i in range(ori_len)
+            if cfg['model_list']['train'][i]['priority'] <= args.priority
+        ]
+        num_train_task = len(model_cfg_train)
+        num_train_win = math.ceil(num_train_task / args.panes_per_window)
+    else:
+        num_train_task = 0
+        num_train_win = 0
+
+    # create a new tmux session
+    os.system(f'tmux new -s {args.session_name} -d')
+
+    # tmux select-window -t 0
+    os.system('tmux select-window -t 0')
+
+    num_infer_task_tmp = num_infer_task
+    num_train_task_tmp = num_train_task
+
+    # create new windows
+    for i in range(num_infer_win):
+        # tmux select-window -t 0
+        os.system('tmux select-window -t 0')
+        # tmux new-window -n infer_1
+        os.system(f'tmux new-window -n infer_{i+1}')
+
+        if num_infer_task_tmp >= args.panes_per_window:
+            num_cur_task = args.panes_per_window
+            num_infer_task_tmp -= args.panes_per_window
+        else:
+            num_cur_task = num_infer_task_tmp
+
+        # split each window into different panes
+        for j in range(num_cur_task - 1):
+            ratio = int(100 - 100 / (num_cur_task - j))
+            os.system(f'tmux split-window -h -p {ratio:02d}')
+
+        os.system('tmux select-layout tiled')
+
+    for i in range(num_train_win):
+        # tmux select-window - t 0
+        os.system('tmux select-window -t 0')
+        # tmux new-window -n train_1
+        os.system(f'tmux new-window -n train_{i+1}')
+
+        if num_train_task_tmp >= args.panes_per_window:
+            num_cur_task = args.panes_per_window
+            num_train_task_tmp -= args.panes_per_window
+        else:
+            num_cur_task = num_train_task_tmp
+
+        # split each window into different panes
+        for j in range(num_cur_task - 1):
+            ratio = int(100 - 100 / (num_cur_task - j))
+            os.system(f'tmux split-window -h -p {ratio:02d}')
+
+        os.system('tmux select-layout tiled')
+
+    port = args.port
+    # select window and pane to run
+    if num_infer_task != 0:
+        num_infer_task_tmp = num_infer_task
+        for i in range(num_infer_win):
+            os.system('tmux select-window -t 0')
+            # tmux select-window -t infer_j
+            os.system(f'tmux select-window -t infer_{i+1}')
+
+            if num_infer_task_tmp >= args.panes_per_window:
+                num_cur_task = args.panes_per_window
```

Comment:
What does 'cur' mean?

Reply:
It means the number of tasks that will be run in the  'current'  window.